### ISSUE TITLE
Blazerunner infinite knock back

### ISSUE THREAD

**Issue by baranzadeoglu** [chromiecraft/chromiecraft#1387]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

instant and constant knock backs (quite absurd in amount)

### Expected Blizzlike Behaviour

Hard to assess. Classic DB has no reference for a knock back skill but comments include varying accounts. I can confirm there is something wrong with the mob, but not how it is supposed to be.

### Source

https://classic.wowhead.com/npc=9376/blazerunner#comments
https://youtu.be/vaJ8lHIt3Nk?t=46
classic wow video of the engagement.
### Steps to reproduce the problem

Go to Ungoro
49.6 49.6
Engage Blazerunner
Observe

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/36a1777b99c400484537c358af8fa6b3a42b13c7

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-CustomWorldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-TempAnnouncements](https://github.com/55Honey/Acore_TempAnnouncements)

### Customizations

None

### Server

ChromieCraft

**Comment by Azcobu** [comment]
Thanks for the report. I can confirm, having tested it, that the NPC's knockback is pretty insane. This is due to an aura he casts on himself on spawn, [spell 13913](https://wowgaming.altervista.org/aowow/?spell=13913). This aura in turn periodically triggers [spell 13914](https://wowgaming.altervista.org/aowow/?spell=13914) once per second, which performs the knockback. It also gives him 90% damage resistance.

However, the trick with this NPC is that the aura is meant to be dispelled by the [Silver Totem of Aquementas](https://classic.wowhead.com/item=11522/silver-totem-of-aquementas), which you can make from components given to you on the quest. The Youtube link you provided shows the orc warrior actually using the totem to remove his aura (the blue vertical spell effect) which is why he's much more manageable there.

So in summary, I think the ridiculous knockback and the 90% damage resist are intended as indicators that he's not to be messed with without the Silver Totem, rather than an actual bug.
